core/ts/utils/paths.ts

```diff
@@ -7,6 +7,8 @@
  * https://developers.google.com/open-source/licenses/bsd
  */
 
+import { deepUndefinedToNull } from "./object";
```

Comment:
is this intentional? doesn't seem to be used

Reply:
Ty, no, that's something the IDE injected :(